starwars/src/App.js

```diff
@@ -30,9 +31,11 @@ class App extends Component {
   };
 
   render() {
+    console.log(this.state.starwarsChars)
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+      {this.state.starwarsChars.map(item => <Cards key={item} cardsProp={item}/>)}
       </div>
```

Comment:
good job mapping and returning a custom component here. this is the pattern that we will be using in React a lot so you will get very comfortable with it. I would rename the Component to something more applicable liek maybe CharacterCard, but Card isnt incorrect either, just not as clear as to what it is. You could also take it one step further and  extract the map into a component that is just in charge of mapping the state object and returning the Card component, so like a CharacterList component. This is getting into a more functional programming paradigm of having a function only do one thing.